RELEASE-NOTES.md

```diff
@@ -1,3 +1,87 @@
+# Riak CS 1.5.4 Release Notes
+
+## Bugs Fixed
+
+- Disable previous riak object after backpressure sleep is triggered
```

Comment:
@lucperkins  Riak CS internally caches a manifest object in single request-response scope.
By this change it discards the cache and re-read manifests.
So disable means discard cached riak object.